common.mk

```diff
@@ -233,6 +237,8 @@ $(TOP_SMEMS_CONF) $(MODEL_SMEMS_CONF) &:  $(MFC_SMEMS_CONF) $(MFC_MODEL_HRCHY_JS
 		--model-module-name $(MODEL) \
 		--out-dut-smems-conf $(TOP_SMEMS_CONF) \
 		--out-model-smems-conf $(MODEL_SMEMS_CONF)
+#	for blackboxed SRAMs: define mem here and use generated module name in blackbox source
+	-[ -f $(base_dir)/vlsi/add.mem.conf ] && cat $(base_dir)/vlsi/add.mems.conf >> $(TOP_SMEMS_CONF)
```

Comment:
I think there's a cleaner way to do this.  What if you have the generator emit a supplementary file `.custom.mems.conf` via `addResource` in rocket-chip? Then in this make job, concatenate that file into `TOP_SMEMS_CONF` if it exists.

I don't like having this magic file in the vlsi/ directory.